<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtActionButtonsUi</class>
 <widget class="QGroupBox" name="QtActionButtonsUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>94</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>x</string>
  </property>
  <property name="title">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="actionButton1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>New Recipe</string>
     </property>
     <property name="icon">
      <iconset resource="maimage.qrc">
       <normaloff>:/images/NewRecipe</normaloff>:/images/NewRecipe</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="actionButton2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>Paste</string>
     </property>
     <property name="icon">
      <iconset resource="maimage.qrc">
       <normaloff>:/images/Paste</normaloff>:/images/Paste</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="actionButton3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="actionButton4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>Publish</string>
     </property>
     <property name="icon">
      <iconset resource="maimage.qrc">
       <normaloff>:/images/Go</normaloff>:/images/Go</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="actionButton5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>64</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionNewRecipe">
   <property name="icon">
    <iconset resource="maimage.qrc">
     <normaloff>:/images/NewRecipe</normaloff>:/images/NewRecipe</iconset>
   </property>
   <property name="text">
    <string>New Recipe</string>
   </property>
   <property name="toolTip">
    <string>Create New Recipe</string>
   </property>
  </action>
  <action name="actionPublish">
   <property name="icon">
    <iconset resource="maimage.qrc">
     <normaloff>:/images/Go</normaloff>:/images/Go</iconset>
   </property>
   <property name="text">
    <string>Publish</string>
   </property>
   <property name="toolTip">
    <string>Publish Recipe</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="maimage.qrc"/>
 </resources>
 <connections/>
</ui>
